bundle/edu.gemini.qpt.client/src/main/java/edu/gemini/qpt/ui/view/property/adapter/ObsAdapter.java

```diff
@@ -37,7 +37,7 @@ static void addObsProperties(Variant variant, Obs target, PropertyTable table) {
 		table.put(PROP_TITLE, target.getTitle());
 		table.put(PROP_INSTRUMENT, target.getInstrumentStringWithConfig());
 		table.put(PROP_CONSTRAINTS, target.getConstraintsString());
-		table.put(PROP_COORDINATES, target.getCoords());
+		table.put(PROP_COORDINATES, target.getCoords(variant.getSchedule().getMiddlePoint()));
```

Comment:
Correct. We also want the coordinates to be updated when dragging the observation as @andrewwstephens mentioned. I think I already know how to do it.